docs/debugging/README.md

```diff
@@ -3,7 +3,7 @@ navGroup: Support
 navTitle: Debugging Node-RED issues
 meta:
   description: Learn how to troubleshoot unresponsive Node-RED instances using Safe Mode.
-  tags:
+   tags:
```

Comment:
I think this one didn't need the extra space

Reply:
done !